nuxt/plugin.js

```diff
@@ -0,0 +1,4 @@
+import Vue from 'vue';
+import VueCarousel from 'vue-carousel/dist/vue-carousel-ssr.min.js';
```

Comment:
I'm not sure is it good assumption b/c maybe someone wants to use client-side bundle in nuxt or maybe I should do not take it as an issue for me